transforms/tabular/tabular-merger-plugin/src/polus/plugins/transforms/tabular/tabular_merger/__main__.py

```diff
@@ -0,0 +1,112 @@
+"""Tabular Merger."""
+import json
+import logging
+import pathlib
+import time
+from typing import Any, Optional
+
+import filepattern as fp
+import typer
+
+from polus.plugins.transforms.tabular.tabular_merger import tabular_merger as tm
+
+app = typer.Typer()
+
+# Initialize the logger
+logging.basicConfig(
+    format="%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s",
+    datefmt="%d-%b-%y %H:%M:%S",
+)
+logger = logging.getLogger("polus.plugins.transforms.tabular_merger")
```

Comment:
log level should be set to the value in the environment variable, and default to `logging.INFO`.

Reply:
done